README.rst

```diff
@@ -0,0 +1,180 @@
+.. ## Define a hard line break in HTML
+.. |br| raw:: html
+
+  <br />
+
+.. # Add the reference for the badges
+.. ## Reference to the CI status
+.. |Linter&Tests| image:: https://github.com/mind-inria/hidimstat/actions/workflows/ci.yml/badge.svg?branch=main
+   :target: https://github.com/mind-inria/hidimstat/actions/workflows/ci.yml?query=branch%3Amain
+.. |CircleCI/Documentation| image:: https://circleci.com/gh/mind-inria/hidimstat.svg?style=shield
+   :target: https://circleci.com/gh/mind-inria/hidimstat?branch=main
+.. |CodeCov| image:: https://codecov.io/github/mind-inria/hidimstat/branch/main/graph/badge.svg?token=O1YZDTFTNS
+   :target: https://codecov.io/github/mind-inria/hidimstat
+.. ## Distribution python
+.. |PyPi| image:: https://img.shields.io/pypi/v/hidimstat.svg
+   :target: https://pypi.org/project/hidimstat/
+.. |PyPi_download| image:: https://img.shields.io/pypi/dm/hidimstat
+   :target: https://pypi.org/project/hidimstat/
+.. |PythonVersion| image:: https://img.shields.io/pypi/pyversions/hidimstat.svg?color=informational
+   :target: https://pypi.org/project/hidimstat/
+.. |Latest release| image:: https://img.shields.io/github/release/mind-inria/hidimstat.svg?color=brightgreen&label=latest%20release
+.. ## Additional badge
+.. |License| image:: https://img.shields.io/github/license/mind-inria/hidimstat
```

Comment:
```suggestion
```
In line with my suggestions to make the readme lighter: the license is already described in the "about" section, visible on the same webpage.